### ISSUE TITLE
rxjs-no-unsafe-switchmap doesn't seem to work

### ISSUE THREAD

**Issue by seaBubble** [cartant/rxjs-tslint-rules#50]
I have set the rxjs-no-unsafe-switchmap to true in the tslint.json file of my Angular project, but when I run ng lint I get no errors. I initially configured it as:

"rules": {
  "rxjs-no-unsafe-switchmap": {
    "options": [{
      "disallow": ["add", "create", "delete", "post", "put", "remove", "set", "update"],
      "observable": "action(s|\\$)?"
    }],
    "severity": "error"
  }
}

But to no avail, so I removed everything else from my tslint file and left only:

```
 {
     "extends": [
         "rxjs-tslint-rules"
     ],
     "rules": {
         "rxjs-no-unsafe-switchmap": {
             "options": [{
                 "disallow": "."
             }],
             "severity": "error"
         },
         "rxjs-no-create": true
     }
 }
```

Running ng lint does detect errors related to the rxjs-no-create rule, but nothing related to switchmaps.

I have, in my effects:

```
 @Effect()
     deleteTest = this.actions
         .pipe(
             ofType(testActions.type.DELETE_TEST),
             map((action: testActions.DeleteTestAction) => action.payload),
             switchMap((payload: { id: number }) =>
                 this.cs.deleteTest(payload.id)
                     .pipe(
                         map(() =>
                             new testActions.DeleteTestSuccessAction({ id: payload.id })
                         ),
                         catchError(() => of(new testActions.DeleteTestFailureAction()))
                     )
             )
         );
```

As I understand it, this should be displaying a lint error.

**Comment by cartant** [comment]
This should be fixed in 4.4.3.
